OxfordBodleian/Juel-Jensen/BDLaethf32.xml

```diff
@@ -124,51 +154,128 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <desc>A white sticker is glued on <locus target="#1r"/>. The name of <persName ref="PRS5782JuelJen"/> is printed on it
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signature "MS. Aeth. f. 32" is written in pencil.
+                                    </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc>Some writings have been added in pencil on <locus target="#1r"/> by the hand of <persName ref="PRS5782JuelJen"/>. On the upper margin is written
+                                        an amount of money which presumably indicates the price of sale of the manuscript <q xml:lang="en">"£ 120"</q>; below are found
+                                        information on the dating and the decorations <q xml:lang="en">"C 18 - 2 miniatures"</q>, as well as the signature of the book <q xml:lang="en">"Ms Aeth. J-J 39"</q>.
+                                        Furthermore, the title of the book in English <q xml:lang="en">"Praise for the Holy Communion of Our Mother Mary written 
+                                            <supplied reason="omitted">by</supplied> Aba Hariyaqos of Bhansa"</q> is written on the lower margin of <locus target="#1r"/>.
+                                    </desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc>
+                                        Corrections are infrequent. They mostly consist of interlineal additions (e.g. <locus target="#17r #21v #65v"/>) and corrections through dashes above and below 
+                                        the letters (e.g. <locus target="#15r #43v #75r"/>).
+                                    </desc>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus from="1r" to="1v"/>
+                                    <locus from="82v" to="84r"/>
+                                    <desc>Scribbles and pen trials in blue or violet ink.</desc> 
+                                </item>
+                                <item xml:id="e5">
+                                    <locus target="#3v"/> 
+                                    <locus from="86r" to="86v"/>
+                                    <desc>Crude drawings of human figures in red ink or blue pencil.</desc> 
+                                </item>
+                                <item xml:id="e6">
+                                    <locus from="81v" to="82r"/>
+                                    <desc>Several names of individuals have been added in blue pencil or violet ink in a very inaccurate handwriting.</desc> 
```

Comment:
I think it would be better to create records for those whose names are legible. I know it's a bit annoying, but one can never know if they might appear together also in another manuscript....

Reply:
Thank you for this suggestion. The records for the legible names will come soon.